============================= test session starts ==============================
platform sunos5 -- Python 3.7.13, pytest-6.2.5, py-1.11.0, pluggy-0.13.1
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
Using --randomly-seed=XXXXXXXXXX
rootdir: $(SOURCE_DIR)
plugins: cov-3.0.0, reporter-0.5.1, mock-3.7.0, asyncio-0.18.1, timeout-2.0.2, benchmark-3.4.1, flaky-3.7.0, randomly-3.11.0, hypothesis-4.57.1
asyncio: mode=legacy
collected 8 items

tests.py ........                                                        [100%]

=============================== warnings summary ===============================
../../../../../../../usr/lib/python3.7/vendor-packages/pytest_asyncio/plugin.py:191
  /usr/lib/python3.7/vendor-packages/pytest_asyncio/plugin.py:191: DeprecationWarning: The 'asyncio_mode' default value will change to 'strict' in future, please explicitly use 'asyncio_mode=strict' or 'asyncio_mode=auto' in pytest configuration file.
    config.issue_config_time_warning(LEGACY_MODE, stacklevel=2)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
========================= 8 passed, 1 warning in X.XXs =========================
